piker/data/_source.py

```diff
@@ -254,61 +253,6 @@ def iterfqsns(self) -> list[str]:
         return keys
 
 
-def from_df(
```

Comment:
ahh yeah right!

we can drop this (and maybe `pandas` altogether for now) since i rewrote the `ib` ohlc frame parser to just cast directly to `numpy`.